internal/graphics/vertex.go

```diff
@@ -16,6 +16,8 @@ package graphics
 
 const (
 	ShaderSrcImageCount = 4
+	// The number of destination images is 1 so far, but this might change in the future.
```

Comment:
Done!
I don't know if you noticed, but I originally added it here: https://github.com/hajimehoshi/ebiten/pull/3013/commits/8abc922e89f49f9dd703d0a8819cf9f76e17f783#diff-58093df023b3ec55c1b71cedc407ec23a6bdce12c442127e3e0250968cdf260aR333

I decided to leave it there as well, since it's kind of a higher level API and thought the extra explanation was relevant